docs/src/pages/presets/writing-presets/index.md

```diff
@@ -99,6 +101,31 @@ module.exports = {
 };
 ```
 
+### Addons
+
+For users, the name `managerEntries` might be a bit too technical, so instead both users and preset-authors can simply use the property: `addons`:
+
+```js
+module.exports = {
+  addons: ['@storybook/addon-storysource/register'],
+};
+```
+
+The array of values can support both references to other presets and addons that should be included into the manager.
+
+This was decided so users didn't have to learn more concepts and the minor differences. Storybook will detect whether the module references a preset or an managerEntry and will do the appropriate thing.
```

Comment:
Let's be explicit about what storybook does here rather than handwavey:

```suggestion
Storybook will automatically detect whether a reference to an addon is a preset or a manager entry by checking if the package contains a `./preset.js` or `./register.js` (manager entry), falling back to preset if it is unsure.

If this heuristic is incorrect for an addon you are using, you can explicitly opt in to an entry being an a manager entry using the `managerEntries` key.
```